### PR TITLE
Add shell (sh)

### LINKED ISSUE #166: Add sh
Simple Hello World in sh
Closes #156

### PR DESCRIPTION
**Pull request by valkheim**
Adding an hello world example running on sh
Closes #166

### CODE REVIEW

**Review by sbshah97 — CHANGES_REQUESTED** [review]
Reference the Issue by adding the Pull Description.

**Comment by sbshah97** [comment]
Also any particular reasons to put she since Bash is already there.

File: sh/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+## Getting Started
+
```
Comment (sbshah97) [inline sh/README.md:2]: Explain how to setup in Windows, Linux and MacOS. Windows has set up a new bash installer. Make sure you mention about it.

**Comment by valkheim** [comment]
sh is a specification, not an implementation. For some historical reasons it is oftenly a symlink to another shell program. There are multiple reasons why we should always have a sh running.
It is standardized and easy to learn. But most of all, it is portable across posix systems.

File: sh/README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+### Requirements
+
+You have to got a terminal running shell program (eg : /bin/sh)
+
+### Start the program
+
+```
+$> ./hello.sh
```
Comment (sbshah97) [inline sh/README.md]: You need to give permissions to run the file using `chmod +x hello.sh`

File: sh/hello.sh [hunk 3]
```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+echo Hello World
```
Comment (sbshah97) [inline sh/hello.sh]: Use double quotes.

**Review by sbshah97 — CHANGES_REQUESTED** [review]
Make changes accordingly.

**Comment by valkheim** [comment]
Thanks for having noticed these issues

**Review by sbshah97 — CHANGES_REQUESTED** [review]
Hi, kindly reference the Issue that you are resolving and then I'll merge the PR! Make sure you write Closes `#<Issue Number>` in the PR Description!
In case an Issue isn't created, please create that and then send in the PR!

**Comment by sbshah97** [comment]
Any problems here?

**Review by sbshah97 — APPROVED** [review]
Looks good. Well done!

**Pull request merged**